include/wil/registry_basic.h

```diff
@@ -908,20 +672,24 @@ namespace wil
 
                 typename err_policy::result delete_value(_In_opt_ PCWSTR value_name) WI_NOEXCEPT
                 {
-                    return err_policy::HResult(HRESULT_FROM_WIN32(::RegDeleteValueW(::wil::reg::details::get_key(m_key), value_name)));
+                    return err_policy::HResult(HRESULT_FROM_WIN32(::RegDeleteValueW(m_key, value_name)));
                 }
 
             private:
-
                 template <typename R, typename get_value_internal_policy = err_policy>
                 typename get_value_internal_policy::result get_value_internal(_In_opt_ PCWSTR subkey, _In_opt_ PCWSTR value_name, R& return_value, DWORD type = ::wil::reg::reg_view_details::get_value_type<R>()) const
                 {
+                    DWORD bytes_allocated{};

```

Comment:
I'd make this explicitly 0 for clarity

Reply:
I tried to be consistent across the board to use default initializers. I actively avoided writing out initialization sometimes, but not others.